pkg/devfile/parser/content_test.go

```diff
@@ -39,8 +40,12 @@ func TestSetDevfileContent(t *testing.T) {
 	t.Run("valid file", func(t *testing.T) {
 
 		var (
-			tempDevfile = createTempDevfile(t, validJsonRawContent100())
-			d           = DevfileCtx{absPath: tempDevfile.Name()}
+			fakeFs      = filesystem.NewFakeFs()
```

Comment:
shouldn't this test be converted to table format?

Reply:
+1 on this